README.md

```diff
@@ -36,6 +36,16 @@ sudo apt install ./dist/input-remapper-1.4.2.deb
 
 input-remapper is now part of [Debian Unstable](https://packages.debian.org/sid/input-remapper)
 
+##### Fedora
+
+On fedora, you should first install python dependecies via below command and then continue with pip installation!
+
+```bash
+sudo dnf install python3-evdev python3-devel gtksourceview4 python3-pydantic
+sudo dnf install python-pydbus
+sudo dnf install xmodmap
+```
```

Comment:
- python3-evdev from the fedora repo is too old
- python-pydbus does not seem to exist: https://packages.fedoraproject.org/search?query=python-pydbus, did you mean python3-pydbus?

Except for python3-pydbus and xmodmap, which is an optional dependency, all of their exact names are already named in the pip section, so I'm not sure what the benefit of this would be.

I'd like to have a script here that generates a .rpm package file, so that the fedora package manager can properly manage it.

Reply:
Theoretically, using `pip` should be the same. But, in the case of `fedora`, it is not.

Reply:
I think, I should also change `python-evdev` to `python3-evdev` according to [this](https://github.com/sezanzeb/input-remapper/issues/83#issuecomment-1030859666). But, as both of them are old as discussed, we eventually need to run `sudo pip install evdev`.